spec/features/dams_object_spec.rb

```diff
@@ -505,4 +505,9 @@
     expect(page).to have_selector('button#node-btn-1:first',:text=>'Parameters')
     page.should_not have_css("button#node-btn-1", :count => 2)
   end
+  it "should not see label Creation Date and Issued Date" do
```

Comment:
What's the intended outcome of the test here? The text says "should not see Creation Date and Issued Date" but the #expect assertions are showing the opposite. Should the text change to "should see", or should the test assertions change?